### ISSUE TITLE
TonY Dataproc initialization fails 

### ISSUE THREAD

**Issue by gogasca** [linkedin/TonY#232]
For some reason Internet access is not available during VM creation, and we can't build the .jar file. 

Workaround: Retry build manually
```
cd /opt/tony/TonY
./gradlew build -x test
cp /opt/tony//TonY/tony-cli/build/libs/tony-cli-*-all.jar /opt/tony//TonY-samples
```

```
FAILURE: Build failed with an exception.
* What went wrong:
Could not determine the dependencies of task ':tony-cli:shadowJar'.
> Could not resolve all dependencies for configuration ':tony-cli:runtimeClasspath'.
   > Could not resolve org.apache.hadoop:hadoop-annotations:2.7.3.
     Required by:
         project :tony-cli > project :tony-core > org.apache.hadoop:hadoop-common:2.7.3
         project :tony-cli > project :tony-core > org.apache.hadoop:hadoop-yarn-api:2.7.3
         project :tony-cli > project :tony-core > org.apache.hadoop:hadoop-yarn-client:2.7.3
         project :tony-cli > project :tony-core > org.apache.hadoop:hadoop-yarn-common:2.7.3
         project :tony-cli > project :tony-core > project :tony-mini > org.apache.hadoop:hadoop-yarn-server-tests:2.7.3
         project :tony-cli > project :tony-core > project :tony-mini > org.apache.hadoop:hadoop-yarn-server-tests:2.7.3 > org.apache
.hadoop:hadoop-yarn-server-common:2.7.3
         project :tony-cli > project :tony-core > project :tony-mini > org.apache.hadoop:hadoop-yarn-server-tests:2.7.3 > org.apache
.hadoop:hadoop-yarn-server-nodemanager:2.7.3
```

https://stackoverflow.com/questions/25243342/gradle-build-is-failing-could-not-resolve-all-dependencies-for-configuration

**Comment by erwa** [comment]
Hi @gogasca ,
I've published tony-cli 0.3.1 with my #230 fix this morning to Maven here: https://repo1.maven.org/maven2/com/linkedin/tony/tony-cli/0.3.1/

It should be searchable on https://search.maven.org/ in a few hours. You can give that a shot once it's available.

**Comment by gogasca** [comment]
Thanks! Not able to reproduce it. Closing
